authors/urls.py

```diff
@@ -18,9 +18,9 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-
-    url(r'^api/', include('authors.apps.authentication.urls',
-                          namespace='authentication')),
+    url(r'^api/', include('authors.apps.authentication.urls', namespace='authentication')),
+    url(r'^articles/', include('authors.apps.articles.urls', namespace='articles')),
```

Comment:
Any reason why you omitted the `api/` part from the articles url? I think that should be included in all urls

Reply:
Done